tdx/issuer.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright (c) Edgeless Systems GmbH
```

Comment:
I believe this is not from Edgeless, can be removed

Reply:
both have been copied and modified from `internal/attestation/tdx`